JitsiMeetJS.js

```diff
@@ -42,6 +43,9 @@ var LibJitsiMeet = {
 
     version: '{#COMMIT_HASH#}',
 
+    // expose JitsiTrackError this way to give library consumers to do checks
+    // like if (error instanceof JitsiMeetJS.JitsiTrackError) { }
+    JitsiTrackError: JitsiTrackError,
```

Comment:
@hristoterezov @yanas PRs about JitsiTrackError refactoring - https://github.com/jitsi/lib-jitsi-meet/pull/149 and https://github.com/jitsi/jitsi-meet/pull/707